configure.ac

```diff
@@ -416,9 +416,9 @@ LOC_ARG_WITH_LDFLAGS(cairo, cairo)
 LOC_ARG_WITH_INC(freetype, FreeType)
 LOC_ARG_WITH_LIB(freetype, FreeType)
 
-LOC_ARG_WITH_INC(proj, External PROJ.4)
-LOC_ARG_WITH_LIB(proj, External PROJ.4)
-LOC_ARG_WITH_SHARE(proj, External PROJ.4)
+LOC_ARG_WITH_INC(proj, External PROJ)
```

Comment:
I suppose we can remove `External`, it is just a remnant from GRASS 4-5, when PROJ was embedded, isn’t it?

Reply:
No opinion on this